framework/dbt/basic/macros/overrides.sql

```diff
@@ -26,15 +26,7 @@
 
 {%- endmacro %}
 
-{% macro postgres__create_schema(relation) -%}
-  {%- call statement('create_schema') -%}
-    /* schemas are not created in CrateDB */
-    DROP TABLE IF EXISTS thisschemadefinitelydoesnotexits.thiswouldnotexist
-    /* but we need to run something to not have just EOF */
-  {% endcall %}
-{% endmacro %}
-
-{% macro postgres__create_table_as(temporary, relation, sql) -%}
+{% macro cratedb__create_table_as(temporary, relation, sql) -%}
```

Comment:
Would not this override be part of `dbt-cratedb2`?

Reply:
> If you are good with the new variant?

Those are the corresponding overrides currently shipping with dbt-cratedb2, very closely derived from dbt-postgres, with only necessary amendments applied:
https://github.com/crate/dbt-cratedb2/blob/v0.0.2/dbt/include/cratedb/macros/adapters.sql#L1-L40

In this case, when compared with PostgreSQL, the macro just omits the `TEMPORARY` modifier in `CREATE TEMPORARY TABLE` statements.

Reply:
> the macro just omits the `TEMPORARY` modifier

This is certainly an important detail that should go into the "caveats" section of the documentation.

- https://github.com/crate/dbt-cratedb2/issues/3

Reply:
With the most recent update, I've removed all macros here.